covid_graphs/covid_graphs/scripts/country_dashboard.py

```diff
@@ -46,17 +46,14 @@ def __init__(self, dashboard_type: DashboardType, data_dir: Path, server: Flask)
         ]
         self.report_by_short_name = {report.short_name: report for report in reports}
 
-        if (
-            dashboard_type == DashboardType.SingleCountry
-            or dashboard_type == DashboardType.AllCountries
-        ):
-            self.graph_dict = {
-                prediction_event.name: CountryDashboard._create_graphs(
-                    self.report_by_short_name, prediction_event
-                )
-                for prediction_event in prediction_db.get_prediction_events()
-            }
+        self.graph_dict = {
```

Comment:
What do you think about `self.graphs_by_event`?

Reply:
Done